src/node/nodetonode.h

```diff
@@ -41,6 +41,16 @@ namespace ccf
         signed_public.value());
     }
 
+    bool is_channel_established(NodeId id, Channel& channel)
```

Comment:
The name of the function does not describe what it does. It does check if the channel is established but then it will attempt to establish a channel if it is not.

Reply:
True but we already have an establish() method. I’ll try to come up with something more descriptive

Reply:
going with `try_established_channel`